sql/src/test/java/io/crate/integrationtests/TableFunctionITest.java

```diff
@@ -53,6 +54,7 @@ public void testSelectFromUnnestWithScalarFunction() {
     }
 
     @Test
+    @UseJdbc(1) // otherwise, missing type information
```

Comment:
not sure how to handle `insert into t (select * from unnest(?, ?))` when it is executed without param type info, shall we we adjust param types in RelationAnalyzer#visitTableFunction` as it is done in `RelationAnalyzer#visitValues`?

Reply:
Shouldn't it be possible to infer the types for the parameters based on the target table columns?

Reply:
I try to use the `StatementContext` to infer the types of the parameter symbols, but now `RelationAnalyzer#visitTableFunction` has a special handling for the `UnnestTable` function.